tests/core/test_agent.py

```diff
@@ -194,6 +201,22 @@ def test_agent_wrong_use_of_load(tmpdir, default_domain):
     assert agent.model_directory is not None
 
 
+def test_init_agent_with_missing_policies():
```

Comment:
use https://docs.pytest.org/en/latest/parametrize.html

Reply:
But what would I parametrize here?

Reply:
the domain content, and the policy configuration.

Reply:
So I'd either add the string for the domain/config inside the method OR parameterize, but not both, correct?

Reply:
yes